src/esp32/entry.rs

```diff
@@ -95,10 +97,6 @@ pub async fn serve_web_inner(
             (cfg_response, robot)
         };
 
-        let address: SocketAddr = "0.0.0.0:12346".parse().unwrap();
-        let tls = Box::new(Esp32Tls::new_server(&tls_server_config));
-        let tls_listener = Esp32Listener::new(address.into(), Some(tls)).unwrap();
```

Comment:
Is `Esp32Listener` now unused? Any other things that become unused if that thread is pulled?

Reply:
not really the code is shared with the client socket